docs/UserGuide.md

```diff
@@ -112,6 +112,35 @@ Email:
 Please provide a valid index.
 ```
 
+#### Command: `deleteCompany`
+
+The **deleteCompany** command allows you to remove companies from your Connectify database. To delete a company, you need to specify the company's index.
```

Comment:
Thanks for adding these to the UG! Users will have a better understanding of what commands there are at their disposal.